components/esm/loadland.hpp

```diff
@@ -110,7 +110,7 @@ struct Land
         static void transposeTextureData(const uint16_t *in, uint16_t *out);
     };
 
-    void load(ESMReader &esm, bool &isDeleted);
+    virtual void load(ESMReader &esm, bool &isDeleted);
```

Comment:
apps/opencs/model/world/land.hpp line 18 has this exact method in derived class. Without virtual this method would be shadowed. Adding virtual will allow for dynamic dispatch / polymorphism / method overriding. Right?